README.md

```diff
@@ -29,41 +30,38 @@ npm install zaproxy
 
 ## Usage
 
-By default ZAP [requires an API key](https://www.zaproxy.org/faq/why-is-an-api-key-required-by-default/) to be sent with every request. This is done automatically providing you supply the same API key when you instantiate the `ZapClient` that you use to run ZAP with. All following API requests will use this same API key.  
+By default ZAP [requires an API key](https://www.zaproxy.org/faq/why-is-an-api-key-required-by-default/) to be sent with every request. This is done automatically providing you supply the same API key when you instantiate the `ZapClient` that you use to run ZAP with. All following API requests will use this same API key.
 You can disable the API key when running ZAP if you are on a trusted network and understand the risks. If you choose to do so, you may omit the `apiKey` property of the `zapOptions` object supplied to `ZapClient`.
 
 ### Instantiate the Node API:
 
 ```js
 const ZapClient = require('zaproxy');
 
-const zapOptions = {
-  apiKey: <the key you supplied to ZAP when you started it>, // I.E. 'v90dnblpqs1pcac991tn2oudl'
-  proxy: <protocol>://<host>:<port> // I.E. 'http://192.168.0.10:8080'
-};
-
-const zaproxy = new ZapClient(zapOptions);
+async function invokeApi() {
+  const zapOptions = {
+    apiKey: 'eahhr6h6kal92j21gkcnhkp80t',
+    proxy: {
+      host: '127.0.0.1',
+      port: 8080,
+    },
+  };
+  const zaproxy = new ZapClient(zapOptions);
 ```
 
 ### Use the Node API:
 
-Callbacks:
-
 ```js
-zaproxy.spider.scanAsUser(contextId, userId, sutBaseUrl, maxChildren, recurse, subtreeonly, (err, resp) => {
-  if (err) // Handle the error.
-  if (resp) // Handle the response.
-});
-```
-
-Promises:
-
-```js
-await zaproxy.spider.scanAsUser(contextId, userId, sutBaseUrl, maxChildren)
-  .then(
-    resp => console.log(JSON.stringify(resp)),
-    err => `Error occurred while attempting to scan as user. Error was: ${err.message}`
-  );
+let params = {
+  contextId,
```

Comment:
Some of the parameters don't have the correct case, they should be either mapped or corrected.